### ISSUE TITLE
gamma

### ISSUE THREAD

**Issue by zhly0** [clcarwin/sphereface_pytorch#15]
Thanks for your code!
but the code:
loss = -1 * (1-pt)**self.gamma * logpt
in the Angleloss,I really did not get it,because,the default gamma is 0,and the term:
(1-pt)**self.gamma is useless,it is equal to:
loss = -1 * logpt
my question is , is 0 is your default gamma value?

**Comment by clcarwin** [comment]
Yes, zero is the default value. The code is ```Focal Loss```, you can read the focal loss paper to known more.

**Comment by zhly0** [comment]
Thanks!
